.github/workflows/publish-helm-chart.yaml

```
@@ -8,27 +8,51 @@ on:
         required: false
         type: string
       production:
-        description: Push to production registry (default dev)
+        description: Push to production registry (default dev; GHCR only)
         required: false
         type: boolean
         default: false
+      registry:
+        description: Target OCI registry (ghcr = GHCR, distr = Distr pilot)
+        required: false
+        type: choice
```

actionlint (1.7.12)

[error] invalid value "choice" for input type of workflow_call event. it must be one of "boolean", "number", or "string"

(syntax-check)